cfme/tests/services/test_rest_services.py

```diff
@@ -778,3 +778,77 @@ def test_edit_orchestration_templates(self, rest_api, orchestration_templates, f
             assert edited[i].description == new[i]['description']
             orchestration_templates[i].reload()
             assert orchestration_templates[i].description == new[i]['description']
+
+    @pytest.mark.tier(3)
+    @pytest.mark.uncollectif(lambda: version.current_version() < '5.8')
+    @pytest.mark.parametrize(
+        "from_detail", [True, False],
+        ids=["from_detail", "from_collection"])
+    def test_copy_orchestration_templates(self, request, rest_api, orchestration_templates,
+            from_detail):
+        """Tests copying of orchestration templates.
+
+        Metadata:
+            test_flag: rest
+        """
+        response_len = len(orchestration_templates)
+        new = []
+        for _ in range(response_len):
+            uniq = fauxfactory.gen_alphanumeric(5)
+            new.append({
+                "name": "test_copied_{}".format(uniq),
+                "content": "{ 'Description' : " + "'{}'".format(uniq) + "}\n"
+            })
+        if from_detail:
+            copied = []
+            for i in range(response_len):
+                copied.append(orchestration_templates[i].action.copy(**new[i]))
+                assert rest_api.response.status_code == 200
+        else:
+            for i in range(response_len):
+                new[i].update(orchestration_templates[i]._ref_repr())
+            copied = rest_api.collections.orchestration_templates.action.copy(*new)
+            assert rest_api.response.status_code == 200
+
+        request.addfinalizer(
+            lambda: rest_api.collections.orchestration_templates.action.delete(*copied))
+
+        assert len(copied) == response_len
```

Comment:
**Optional**
Consider a soft_assert here so that the tests continues and checks all the records if you don't have the same number.